processing/src/main/java/io/druid/segment/data/GenericIndexed.java

```diff
@@ -53,7 +53,7 @@
  * <p>
  * byte 1: version (0x1)
  * byte 2 == 0x1 =>; allowReverseLookup
- * bytes 3-6 =>; numBytesUsed
+ * bytes 3-6 =>; numBytesUsed, It will be -1 for null values.
```

Comment:
This addition should be at line 59, not here

Reply:
done.